packages/svelte/src/internal/client/dom/reconciler.js

```diff
@@ -30,30 +30,21 @@ export function create_fragment_with_script_from_html(html) {
 
 /**
  * @param {Array<import('../types.js').TemplateNode> | import('../types.js').TemplateNode} current
- * @param {null | Element} parent_element
- * @param {null | Text | Element | Comment} sibling
+ * @param {Text | Element | Comment} sibling
  * @returns {Text | Element | Comment}
  */
-export function insert(current, parent_element, sibling) {
+export function insert(current, sibling) {
```

Comment:
why was it possible to simplify this?

Reply:
resolving as this was discussed this over zoom, no inline comments required